### ISSUE TITLE
Cherry picks for the 1.2 release of DC and DC Virtual Lab

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#667]
Here are the commits to cherry pick for the upcoming 1.2 release:

https://github.com/phetsims/circuit-construction-kit-common/commit/2753793007394244d586a82081b4bd4b3712e36e
https://github.com/phetsims/circuit-construction-kit-common/commit/1dcefb688c4d9c2522e027fb4a3f06323ee93d07
https://github.com/phetsims/circuit-construction-kit-common/commit/24888bc038151ef90be91e801f94f19fa87df08a
https://github.com/phetsims/circuit-construction-kit-dc-virtual-lab/commit/e64b7926eae9fd26d75b13b00615165339d95d21
https://github.com/phetsims/circuit-construction-kit-common/commit/62d165009a235e97584317ff0ab63a7c10ee5604

**Comment by arouinfar** [comment]
@samreid can you add https://github.com/phetsims/joist/commit/991d6a448a869bff9ed7836676eae9fdd2c9aeed to the list? There are joist sounds in the latest rc, which is odd because all other UI sounds are disabled, see https://github.com/phetsims/joist/issues/680.

**Comment by samreid** [comment]
In https://github.com/phetsims/QA/issues/605#issuecomment-773576084 it was said:

> @KatieWoe if you have a chance to start testing before I publish new versions, I think that's OK.  Let's just consider https://github.com/phetsims/joist/commit/d0be22a46c325481ea2cb3bca6bb5d3c02f1a704 as something to be cherry-picked before publication.

In https://github.com/phetsims/QA/issues/605#issuecomment-777985354 @jessegreenberg said:

> @samreid @KatieWoe CCK DC and CCK DC Virtual Lab 1.2 scenery SHAs received a patch for phetsims/phet-android-app#155.
> 
> If possible, you may wish to verify this sim works well in the PhET Android app as part of this test or the next one if there is one.

@jessegreenberg said:

> Sorry @samreid, the regression identified in https://github.com/phetsims/joist/issues/658#issuecomment-772811567 made it to this RC. This reverting commit in joist should be cherry-picked into the joist code for this RC. https://github.com/phetsims/joist/commit/d0be22a46c325481ea2cb3bca6bb5d3c02f1a704

**Comment by samreid** [comment]
I applied patches for all of the above, but @jonathanolson recommended we coordinate with @jessegreenberg about the others before publishing the next RC.

**Comment by samreid** [comment]
> I applied patches for all of the above, but @jonathanolson recommended we coordinate with @jessegreenberg about the others before publishing the next RC.

@pixelzoom indicated that https://github.com/phetsims/joist/issues/680 only affects CCK sims, so I went ahead and patched it.  Afterwards, I realized that it made commits to a branch in joist.  Hopefully this doesn't disrupt ongoing maintenance releases.  Oh, next I realized that the fix for https://github.com/phetsims/phet-android-app/issues/155 was a scenery commit.  I checked that fix was in both scenery branches.  So maybe since it was separate repos they wont interfere.
